avaframe/tests/test_probAna.py

```diff
@@ -47,10 +47,10 @@ def test_probAna(tmp_path):
 
     # call function to test
     pA.probAnalysis(avaDirtmp, cfg, com1DFA, parametersDict=parametersDict, inputDir='')
-    probTest = np.loadtxt(os.path.join(avaDirtmp, 'Outputs', 'ana4Stats', 'avaParabola_probMap1.0.asc'), skiprows=6)
+    probTest = np.loadtxt(os.path.join(avaDirtmp, 'Outputs', 'ana4Stats', 'avaParabola_prob__ppr_lim1.0.asc'), skiprows=6)
```

Comment:
wouldn't this now need to include the probConf?

Reply:
If we'd provide it in the test, then yes, but right now it is not included...